lms/templates/footer.html

```diff
@@ -55,40 +55,39 @@
 
       ## Site operators: Please do not remove this paragraph! This attributes back to edX and makes your acknowledgement of edX's trademarks clear.
       <p class="copyright">
-	## Translators: 'EdX', 'edX', and 'Open edX' are trademarks of 'edX Inc.'. Please do not translate any of these trademarks and company names.
-	${_("EdX, Open edX, and the edX and Open edX logos are registered trademarks or trademarks of {link_start}edX Inc.{link_end}").format(
-	    link_start=u"<a href='https://www.edx.org/'>",
-	    link_end=u"</a>"
-	)}
+        ## Translators: 'EdX', 'edX', and 'Open edX' are trademarks of 'edX Inc.'. Please do not translate any of these trademarks and company names.
+        ${_("EdX, Open edX, and the edX and Open edX logos are registered trademarks or trademarks of {link_start}edX Inc.{link_end}").format(
+            link_start=u"<a href='https://www.edx.org/'>",
+            link_end=u"</a>"
+        )}
       </p>
-        <nav class="nav-legal" aria-label="${_('Legal')}">
-          <ul>
-	    %if marketing_link('HONOR') and marketing_link('HONOR') != '#':
-            <li class="nav-legal-01">
-	      <%
-		 tos_link = u"<a href='{}'>".format(marketing_link('TOS'))
-		 honor_link = u"<a href='{}'>".format(marketing_link('HONOR'))
-	      %>
-	      ${
-	        _("{tos_link_start}Terms of Service{tos_link_end} and {honor_link_start}Honor Code{honor_link_end}").format(
-	          tos_link_start=tos_link,
-	          tos_link_end="</a>",
-	          honor_link_start=honor_link,
-	          honor_link_end="</a>",
-                )
-	      }
-              </a>
```

Comment:
Note that this `</a>` is removed in this pull request, because it is unnecessary. All other changes in this PR are whitespace-only.